packages/desktop-client/src/components/transactions/MobileTransaction.js

```diff
@@ -99,6 +175,768 @@ function Status({ status }) {
   );
 }
 
+// TOOD: export this
+class TransactionEditInner extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      transactions: props.transactions,
+      editingChild: null,
+    };
+  }
+
+  serializeTransactions = memoizeOne(transactions => {
+    return transactions.map(t =>
+      serializeTransaction(t, this.props.dateFormat),
+    );
+  });
+
+  componentDidMount() {
+    if (this.props.adding) {
+      this.amount.focus();
+    }
+
+    document
+      .querySelector('meta[name="theme-color"]')
+      .setAttribute('content', colors.p5);
```

Comment:
Can this be done using `withThemeColor`?

Reply:
It can! I was looking for a hook of that nature and I didn't notice `withThemeColor`, thanks for the callout.

Reply:
Done via https://github.com/actualbudget/actual/pull/1340/commits/d795e8ce0b0d80cdfc65400ce8dbe086db94d4fb.